server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -596,4 +596,13 @@ export default class ServiceProviderReferralsController {
 
     res.render(...view.renderArgs)
   }
+
+  async createDraftEndOfServiceReport(req: Request, res: Response): Promise<void> {
+    const draftEndOfServiceReport = await this.interventionsService.createDraftEndOfServiceReport(
+      res.locals.user.token.accessToken,
+      req.params.id
+    )
+
+    res.redirect(303, `/service-provider/end-of-service-report/${draftEndOfServiceReport.id}/outcomes/1`)
```

Comment:
Just to double-check - this doesn't actually exist yet, right? Or did I miss that PR?

Reply:
Yep, that's right!